components/admin_console/admin_sidebar/admin_sidebar.jsx

```diff
@@ -369,22 +390,15 @@ export default class AdminSidebar extends React.Component {
                                         </OverlayTrigger>
                                     </div>}
                             </li>
-
+                            {this.renderRootMenu(AdminDefinition.about, 'fa-info', 'admin.sidebar.about', 'ABOUT')}
                             {this.renderRootMenu(AdminDefinition.reporting, 'fa-bar-chart', 'admin.sidebar.reports', 'REPORTING')}
-                            <AdminSidebarCategory
-                                sectionClass='sections--settings'
-                                parentLink='/admin_console'
-                                icon='fa-gear'
-                                title={
-                                    <FormattedMessage
-                                        id='admin.sidebar.settings'
-                                        defaultMessage='SETTINGS'
-                                    />
-                                }
-                            >
-                                {Object.values(AdminDefinition.settings).map(this.renderSettingsMenu)}
```

Comment:
You can entirely remove the renderSettingsMenu funcion definition because was used only here.

Reply:
Yep, I'm doing that now actually. Sorry, I tagged you a little prematurely. But good to confirm.